components/typography/Base.tsx

```diff
@@ -524,13 +524,15 @@ class Base extends React.Component<InternalBlockProps, BaseState> {
                     [`${prefixCls}-${type}`]: type,
                     [`${prefixCls}-disabled`]: disabled,
                     [`${prefixCls}-ellipsis`]: rows,
-                    [`${prefixCls}-single-line`]: rows === 1,
+                    // fix flush
+                    [`${prefixCls}-no-wrap`]: rows === 1,
                     [`${prefixCls}-ellipsis-single-line`]: cssTextOverflow,
                     [`${prefixCls}-ellipsis-multiple-line`]: cssLineClamp,
                   },
                   className,
                 )}
                 style={{
+                  width: rows === 1 && '100%',
```

Comment:
Not agree with this change. Maybe `maxWidth` instead.

Reply:
If we use maxWidth here, it will **not** trigger a re-measure. (since the `width` is not changed, the resizeOnNextFrame will not be triggered )
if the width of the parent container grows from small to large, there will be a hidden problem.

![image](https://user-images.githubusercontent.com/5574625/120344511-3a5bbc80-c32c-11eb-9880-78efa3df05fb.png)

Replacing the width attribute directly is not good, It's better to preset by css;

Reply:
在之前的版本中，对于手动指定 ellipsis 的参数，会自动进行剪裁，即使没有默认设置 css width；

这里添加 max-width 后，父容器从小变大时，由于不会触发 resize 逻辑，导致宽度没有展开；

目前根据 ellipsis -> rows: 1 -> nowrap 逻辑，搭配 width 100% 感觉比较贴近之前的计算逻辑。

否则在搭配 ellipsis 时，仍然要手动设置 width: 100%；属于 breaking change